src/Universum.hs

```diff
@@ -201,7 +202,7 @@ unsnoc = foldr go Nothing
        Just (xs, e) -> (x:xs, e))
 
 -- | Lifted version of 'Prelude.print'.
-print :: (X.MonadIO m, PBase.Show a) => a -> m ()
+print :: forall a m . (X.MonadIO m, PBase.Show a) => a -> m ()
```

Comment:
Could you, please, move this function to `Print` module? It's really strange to have `Print` module and not have `print` function there...